sycl/doc/extensions/experimental/SYCL_EXT_ONEAPI_BF16_CONVERSION.asciidoc

```diff
@@ -1,4 +1,4 @@
-= SYCL_INTEL_bf16_conversion
+= SYCL_ONEAPI_bfloat16
```

Comment:
I think your branch must be out-of-date because these specification file names are now all lower case. You should merge the base and resolve conflicts (or maybe it will just be easier to start a new PR).

The name of the file should stay in sync with the feature-test macro.  Since you are renaming the feature-test macro to "SYCL_EXT_ONEAPI_BFLOAT16_CONVERSION", the file name should be "sycl_ext_oneapi_bfloat16_conversion.asciidoc".  The name of the extension (on line 1 of this file) should also be lowercase: "sycl_ext_oneapi_bfloat16_conversion".

However, maybe you are preparing this extension to encompass all bfloat16 features, rather than just conversion?  As I mentioned in #5645, I think it would be better to have a single bloat16 extension like this.  In that case, the name of the feature-test macro should be "SYCL_EXT_ONEAPI_BFLOAT16", the name of the extension should be "sycl_ext_oneapi_bfloat16", and the name of the file should be "sycl_ext_oneapi_bfloat16.asciidoc".

Finally, the header file "include/CL/sycl/feature_test.hpp" should also be updated to define the new feature-test macro, and you should remove the definition of the old feature-test macro.

Reply:
No changes to the extension are in this PR anymore. The extension was updated by #5393. 
So the new aspect is inline with the the extension

Reply:
Thanks. I think this looks good.

@smaslov-intel, do you want a chance to review this from the implementation point of view?